### DISCUSSION TITLE
CI not available to run on one specific PR

### DISCUSSION THREAD

**Discussion by hexcowboy** [community/community#65489] [Actions]
### Select Topic Area

Bug

### Body

https://github.com/hexcowboy/rslock/pull/5

I have CI configured to run on all pull requests, and this contributor has also opened another PR from the same source repository that I was able to run CI on.

As far as I know this isn't a configuration error. Anyone know why I can't run CI on this PR?

**Comment by unknown4now** [comment]
I think your issue is with lifetime. Try this way
> let resource = format!("{}_{}", PREFIX, resource); // Fixed the format string

**Reply by hexcowboy** [reply]
i'm confused on how this relates to CI
